components/accept-financial-contributions/ApplyToHost.js

```diff
@@ -90,7 +90,11 @@ class ApplyToHost extends React.Component {
   }
 
   setTags = tag => {
-    tag === 'all' ? this.setState({ tags: null }) : this.setState({ tags: [tag] });
+    tag === 'all' ? this.setState({ tags: null }) : this.setState({ tags: tag });
```

Comment:
**[:ice_cream: Details & personal preferences]** It confuses me to store a `tag` in a `tags` variable (that, because of the name, I expect to be an array). Also because `HostsContainer` expects an array of strings for this param, doesn't GraphQL complains about that?

---

<details>
<summary>More details about this feedback</summary>
<br/>

This feedback categorization is inspired by [Netlify's Feedback Ladders system](https://www.netlify.com/blog/2020/03/05/feedback-ladders-how-we-encode-code-reviews-at-netlify). It includes 5 levels of feedback:

:rotating_light: **Critical blocker**
This feedback blocks all related work, and requires immediate action. 

:no_entry_sign: **Blocker**
This feedback should block the work from moving forward or being approved, but it does not necessarily require immediate action from the team.

:hammer_and_wrench: **Important**
This feedback should not block the work from moving forward or being approved, but it does require future action from the team. This generally means the feedback is not required for the MVP or initial draft of this work, but should be addressed at some point and we should ideally create an issue to track it.

:sparkles: **Enhancement**
This feedback should not block the work from moving forward or being approved. The project lead or owner should consider the feedback and implement or open an issue it if they agree with it.

:ice_cream: **Details & personal preferences**
This feedback should not block the work from moving forward or being approved. The project lead or owner should consider the feedback and implement or open an issue it if they deem it useful. This is also the category for everything that relates to personal preferences or code style.

</details>

Reply:
I didn't see a complaint from GraphQL about it. I think when I originally named it, I was planning for the case where we would pass an array of tags but we are only doing one at a time at the moment. I can put it back to an array if this makes the most sense and is what GraphQL expects.